blocks/header/header.js

```diff
@@ -38,6 +38,7 @@ async function decorateWebsitePicker(websitePicker) {
   });
 
   const a = document.createElement('a');
+  a.href = '#';
```

Comment:
On sunstar.com, the website picker is not an anchor element. Is there a reason we need to have it as an anchor tag?